MovingBlocks/Terasology
# refactor: merge spawn preview screen into world pre generation screen

### Contains

Merges the screens involved in the advanced game setup, in particular the world setup.

### How to test

1. Start Terasology
2. Go through Advanced Game Setup
3. Check that world can still be started fine
4. Play around with the UI elements to check that there's no crashes or weirdness

### Outstanding before merging

- [X] merge spawn preview screen into world pre generation screen

### Follow-ups

- merge universe setup screen into world pre generation screen
- merge world setup screen into world pre generation screen
- rename world pre generation screen
